src/layers/fp64/geojson-layer/geojson-layer-64.js

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
Assuming this 64 bit version can be deleted once we apply the same configurability to Scatterplot and Path as you have done to PolygonLayer.